res/values/derpquest_strings.xml

```diff
@@ -1183,8 +1183,8 @@
     <string name="volume_panel_oreo">Oreo</string>
     <string name="volume_panel_tiled">Tiled</string>
     <!--  Volume Panel Plugin Values   -->
-    <string name="volume_panel_aosp_val">co.potatoproject.plugin.volume.aosp</string>
-    <string name="volume_panel_compat_val">co.potatoproject.plugin.volume.compact</string>
-    <string name="volume_panel_oreo_val">co.potatoproject.plugin.volume.oreo</string>
-    <string name="volume_panel_tiled_val">co.potatoproject.plugin.volume.tiled</string>
+    <string name="volume_panel_aosp_val" translatable="false">co.potatoproject.plugin.volume.aosp</string>
+    <string name="volume_panel_compat_val" translatable="false">co.potatoproject.plugin.volume.compact</string>
+    <string name="volume_panel_oreo_val" translatable="false">co.potatoproject.plugin.volume.oreo</string>
+    <string name="volume_panel_tiled_val" translatable="false">co.potatoproject.plugin.volume.tiled</string>
```

Comment:
not how you do this. you should not even have the values in the file. the `translatable="false"` is just a label - it does nothing.
```suggestion
```